Frontend/VIAProMa/Assets/Scripts/Analytics/AnalyticsObservibleComponent.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+
+using UnityEngine;
+
+namespace VIAProMa.Assets.Scripts.Analytics.LogTypes
+{
+	public abstract class AnalyticsObservibleComponent<T> : MonoBehaviour, IObservable<T> where T : Logpoint
```

Comment:
- Use 4 spaces for indention. Please, do not infest files with tabs.
- Use `LogTypes` as generic type